drivers/hmis/app/graphql/types/hmis_schema/has_assessments.rb

```diff
@@ -24,23 +24,23 @@ def assessments_field(name = :assessments, description = nil, **override_options
       end
 
       def resolve_assessments_with_loader(association_name = :assessments, **args)
-        load_ar_association(object, association_name, scope: apply_assessment_arguments(Hmis::Hud::Assessment, **args))
+        load_ar_association(object, association_name, scope: scoped_assessments(Hmis::Hud::Assessment, **args))
       end
 
       def resolve_assessments(scope = object.assessments, **args)
-        apply_assessment_arguments(scope, **args)
+        scoped_assessments(scope, **args)
       end
 
       def resolve_assessments_including_wip(scope = object.assessments_including_wip, **args)
-        apply_assessment_arguments(scope, **args)
+        scoped_assessments(scope, **args)
       end
 
       private
 
-      def apply_assessment_arguments(scope, sort_order: nil, role: nil)
+      def scoped_assessments(scope, sort_order: nil, role: nil)
```

Comment:
Were we unable to use the same method name across the various schema files?  I was really hoping `def scoped(` could be consistent.  (Maybe that's not a good thing.)
`def scoped_item` or something would work too if `scoped` is a reserved word.  It just seemed easier to be able to rely on the same method name on all models, but again, maybe that's not a good thing.

Reply:
Ah! So if something imports `has_assessments` and `has_disabilities`, then the methods are are overwritten.  That makes sense, sorry I didn't understand that in the other comment.  

The way we've dealt with that before is to have a helper class rather than a concern, so instead of calling `client.scoped_assessments` you'd have to call HasAssessment.scoped(client, scope)` or something like that.  I think your solution is fine.  Let's go with it for now.